test/forms/buttons/button-area.test.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react'
-import { shallow } from 'enzyme'
 import { ButtonArea } from '../../../src/'
+import { render, screen } from '@testing-library/react'
 
 test('ButtonArea creates div with class "button-area"', () => {
-  const wrapper = shallow(
+  render(
     <ButtonArea>
       <button>Hi</button>
     </ButtonArea>
   )
-  expect(wrapper.find('div').hasClass('button-area')).toBe(true)
-  expect(wrapper.find('button').contains('Hi')).toBe(true)
+
+  expect(screen.getByRole('button')).toBeInTheDocument()
```

Comment:
Is this needed still since we'll have this implicit assertion on line 13?

Reply:
Removed.